kiwi/builder/live.py

```diff
@@ -241,8 +241,8 @@ def create(self):
         # calculate size and decide if we need UDF
         if rootsize.accumulate_mbyte_file_sizes() > 4096:
             log.info('ISO exceeds 4G size, using UDF filesystem')
-            custom_iso_args['create_options'].append('-allow-limited-size')
-            custom_iso_args['create_options'].append('-udf')
+            custom_iso_args['create_options'].append('-iso-level')
+            custom_iso_args['create_options'].append('3')
```

Comment:
mkisofs command has -udf and -UDF flags. genisoimage equivalent to -udf is -UDF, however in mkisofs -udf is a sort of flavour of UDF, so I guess it implies it, I am checking it right now. If that's the case then we could test it by adding this flag as it was before and see if the ouput is UDF. If that's the case we would be in a safer position I guess